dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/UserControllerTest.java

```diff
@@ -117,6 +128,41 @@ void setUp() {
     assertEquals("peter@pan.net", user.getEmail());
   }
 
+  @Test
+  void updateRolesShouldInvalidateUserSessions() {
```

Comment:
ideally this should be broken into 2 tests as it's testing 2 separate flows at once (role change & authority change).
if the test fails it won't be obvious which flow is broken (role change, authority change, or both)